fhir-notification-nats/README.MD

```diff
@@ -0,0 +1,98 @@
+# fhir-notification-nats
+
+The fhir-notification-nats module is designed to publish patient records to a [NATS](http://nats.io) streaming cluster for pub/sub, so published patient records can be consumed by designated downstream applications.  
+
+## Configure IBM FHIR
+In fhir-server/liberty-config/config/default/fhir-server-config.json, find the default NATS configuration shown here:  
+
+```json
+    "nats": {
+        "enabled": false,
+        "cluster": "nats-streaming",
+        "channel": "fhirNotifications",
+        "clientId": "fhir-server",
+        "servers": "nats://nats-node1:4222,nats://nats-node2:4222,nats://nats-node3:4222",
```

Comment:
given the default docker-compose, should this be:
`"servers": "nats://nats-node1:4222,nats://nats-node2:4223,nats://nats-node3:4224",` ?